src/game_api/render_api.hpp

```diff
@@ -68,15 +68,24 @@ struct RenderInfo
     float x_dupe2;
     float y_dupe2;
     uint32_t unknown11;
-    uint32_t unknown12;
-    uint32_t unknown13;
+    uint8_t unknown12a;
+    uint8_t unknown12b;
+    bool unknown12c;
+    bool unknown12d;
+    bool unknown13a;

```

Comment:
unknown13a is true when the entity is culled out, not rendered at all, and get_render_position should revert to get_position

Reply:
well, i think it shouldn't return position at all in this situation, like, it's not rendered
also the function shouldn't return 0.0 when it's wrong entity as that's valid position, but i guess it has to stay to not break compatibility

Reply:
i could potentially write the same stuff as i did for the draw olmec hitbox, so loop thru items until on that is drawn is found and return: it's rendering - offset